### ISSUE TITLE
What is DHTML roadmap techniques (Understanding SC 4.1.2)?

### ISSUE THREAD

**Issue by momdo** [w3c/wcag#289]
https://www.w3.org/TR/2016/NOTE-UNDERSTANDING-WCAG20-20161007/ensure-compat-rsv.html#ensure-compat-rsv-255-head

>Note: This failure may be solved in the future using DHTML roadmap techniques. 

What is the intention of this sentence? I would like to know an external link explained in detail.

**Comment by alastc** [comment]
That's from before my time, but I've not heard of anything about DHTML in the last 5 years, so I've created a PR to remove it (unless someone pipes up about what it is).

**Comment by michael-n-cooper** [comment]
It's a predecessor to ARIA, most likely fine to remove that note. Otherwise it would need to be updated to reference the right part of ARIA if any.
